incubator/jenkins/master-image/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jenkins:2.7.3
+RUN /usr/local/bin/install-plugins.sh kubernetes:0.8 workflow-aggregator:2.3
```

Comment:
We also need

```Dockerfile
&& echo 2.0 > /usr/share/jenkins/ref/jenkins.install.UpgradeWizard.state \
``

Otherwise, your plugin installs may be overridden.